scripts/create-dev-data.js

```diff
@@ -1,14 +1,162 @@
+
+const loremIpsum = `
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+  Nullam fringilla, arcu ut ultricies auctor, elit quam
+  consequat neque, eu blandit metus lorem non turpis.
+  Ut luctus nec turpis pellentesque dignissim. Vivamus
+  porttitor tellus turpis, a tempor velit tincidunt at.
+  Aenean laoreet nulla ut porta semper.
+`.replace(/\s\s+/g, ' ');
+
+const log = msg => `${(new Date()).toUTCString()} INFO [main] - ${msg}`;
+
+async function createData({ githubUsername }) {
+  console.log('Cleaning database...');
+  await cleanDatabase();
+
+  console.log('Creating default action types...');
+  await ActionType.createDefaultActionTypes();
+
+  console.log('Creating users...');
+  const [user1, user2] = await Promise.all([
+    User.create({
+      username: githubUsername,
+      email: `${githubUsername}@example.com`,
+    }),
+
+    User.create({
+      username: 'fake-user',
+      email: 'fake-user@example.com',
+      githubAccessToken: 'fake-access-token',
+      githubUserId: 123456,
+    }),
+  ]);
+
+  console.log('Creating sites...');
+  const [site1, site2] = await Promise.all([
+    siteFactory({
+      demoBranch: 'demo-branch',
+      demoDomain: 'https://demo.example.gov',
+      domain: 'https://example.gov',
+      owner: user1.username,
+      repository: 'example-site',
+      users: [user1, user2],
+    }),
+
+    siteFactory({
+      engine: 'node.js',
+      owner: user1.username,
+      repository: 'example-node-site',
+      users: [user1],
+    }),
+  ]);
+
+  console.log('Creating builds...');
+  const site1Builds = await Promise.all([
+    Build.create({
+      branch: site1.defaultBranch,
+      completedAt: new Date(),
+      source: 'fake-build',
+      state: 'success',
+      site: site1.id,
+      user: user1.id,
+      token: 'fake-token',
+    }),
+    Build.create({
+      branch: site1.defaultBranch,
+      source: 'fake-build',
+      site: site1.id,
+      user: user1.id,
+      token: 'fake-token',
+    }).then(build => build.update({ commitSha: '57ce109dcc2cb8675ccbc2d023f40f82a2deabe1' })),
+    Build.create({
+      branch: site1.demoBranch,
+      source: 'fake-build',
+      site: site1.id,
+      user: user1.id,
+      token: 'fake-token',
+      state: 'error',
+      error: 'Something bad happened here',
+      completedAt: new Date(),
+    }).then(build => build.update({ commitSha: '57ce109dcc2cb8675ccbc2d023f40f82a2deabe2' })),
+  ]);
+
+  const site2Builds = await Promise.all([
+    Build.create({
+      branch: site2.defaultBranch,
+      completedAt: new Date(),
+      source: 'fake-build',
+      state: 'success',
+      site: site2.id,
+      user: user1.id,
+      token: 'fake-token',
+    }),
+    Build.create({
+      branch: 'dc/fixes',
+      source: 'fake-build',
+      site: site2.id,
+      user: user1.id,
+      token: 'fake-token',
+    }).then(build => build.update({ commitSha: '57ce109dcc2cb8675ccbc2d023f40f82a2deabe1' })),
```

Comment:
curious , why is token updated and not part of create object?

Reply:
I dunno, I just copy/pasted it....